ci/Jenkinsfile.android

```diff
@@ -61,7 +66,7 @@ pipeline {
     QT_VERSION = "6.9.2"
     QT_ANDROID_PATH = "/opt/qt/${env.QT_VERSION}/android_arm64_v8a"
     QMAKE = "/opt/qt/${env.QT_VERSION}/android_arm64_v8a/bin/qmake"
-    PACKAGE_TYPE = "${isReleaseBranch ? 'aab' : 'apk'}"
+    PACKAGE_TYPE = "${params.PACKAGE_TYPE ?: (isReleaseBranch ? 'aab' : 'apk')}"
```

Comment:
Wouldn't `PACKAGE_TYPE` type always defined, hence making the `isReleaseBranch` part have no effect?